kore/src/main/scala/org/kframework/definition/outer-ext.scala

```diff
@@ -8,9 +8,8 @@ import org.kframework.attributes._
 import org.kframework.utils.errorsystem.KEMException
 
 import scala.annotation.meta.param
-import scala.collection.Set
+import scala.collection.{Set, mutable}
```

Comment:
I think we agreed stylistically at one point that we don't use this type of import, always one import per line? Not sure.
```suggestion
import scala.collection.Set
import scala.collection.mutable
```

Reply:
Ok. Didn't know about that. IntelliJ did that by default.
I changed my settings so it doesn't happen in the future.